micromasters/settings.py

```diff
@@ -437,6 +437,7 @@ def get_var(name, default):
 STATUS_TOKEN = get_var("STATUS_TOKEN", "")
 HEALTH_CHECK = ['CELERY', 'REDIS', 'POSTGRES', 'ELASTIC_SEARCH']
 
+ADWORDS_CONVERSION_ID = get_var("ADWORDS_CONVERSION_ID", "")
```

Comment:
Should this get added to `app.json` as an optional var so it shows up in the Heroku settings dashboard?